Wikipedia/Code/ArticleLocationCollectionViewController.swift

```diff
@@ -192,8 +192,9 @@ extension ArticleLocationCollectionViewController {
 // MARK: - CollectionViewContextMenuShowing
 extension ArticleLocationCollectionViewController: CollectionViewContextMenuShowing {
     func articleViewController(for indexPath: IndexPath) -> ArticleViewController? {
+        
         let articleURL = articleURL(at: indexPath)
-        let articleViewController = ArticleViewController(articleURL: articleURL, dataStore: dataStore, theme: theme)
+        let articleViewController = ArticleViewController(articleURL: articleURL, dataStore: dataStore, theme: theme, source: .places)
```

Comment:
Looks like this method is only called when you peek into an article. I wonder if we should just use .undefined for peeks?

I also wanted to call out that ArticleLocationCollectionViewController is used in places but also in this Explore card > Nearby list view:
![Screenshot 2025-02-06 at 4 06 01 PM](https://github.com/user-attachments/assets/a9ec5e3e-c926-427a-929a-a57ade72c5a9)
![Screenshot 2025-02-06 at 4 06 17 PM](https://github.com/user-attachments/assets/0809487d-52b2-44e7-9a32-106b7764b146)

So we might be logging as being from places but it's actually from this screen.